libgadget/timestep.c

```diff
@@ -731,11 +731,10 @@ int rebuild_activelist(inttime_t Ti_Current, int NumCurrentTiStep)
     {
         const int bin = P[i].TimeBin;
         const int tid = omp_get_thread_num();
+        if(P[i].IsGarbage)
```

Comment:
Was this crashing the code due to the condition is no longer strictly met? It would be nice to describe how a garbage could still be seen here. (due to a reduced frequency of GC).

Reply:
Exactly, comment added.